### PR TITLE
Getting website ready for 0.11 release

### PR DESCRIPTION
**Pull request by Garyfallidis**
Hi this PR fixes crucial issue #906 and minor issue #900. Also it adds the dki tutorial back in the index.

### CODE REVIEW

File: doc/release0.11.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. _release0.10:
+.. _release0.11:
```
Comment (arokem) [inline doc/release0.11.rst]: Good catch!

Comment (Garyfallidis) [inline doc/release0.11.rst]: Hey after fixed this for some reason the link on 0.10 in the index.rst stopped linking.
I am now rerunning the website to double check.
Finally, we can compile the website and do other work simultaneously!! :8ball:

File: tools/make_examples.py [hunk 2]
```diff
@@ -117,10 +117,12 @@ def show():
...
 
 for script in validated_examples:
+    namespace = {}
     figure_basename = os.path.join('fig', os.path.splitext(script)[0])
-    print script
-    execfile(script)
-    plt.close('all')
+    print(script)
+    execfile(script, namespace)
+    del namespace
```
Comment (arokem) [inline tools/make_examples.py]: Ah yes. That's bonkers...

File: doc/examples/bundle_registration.py [hunk 3]
```diff
@@ -83,6 +86,8 @@ def show_both_bundles(bundles, colors=None, show=False, fname=None):
 
    **After bundle registration**.
 
+As you can see the two cingulum bundles are well aligned although they contain many streamlines of different length and shape.
```
Comment (arokem) [inline doc/examples/bundle_registration.py]: PEP8: long line...

File: doc/examples/bundle_registration.py [hunk 4]
```diff
@@ -22,7 +23,9 @@
 """
 An important step before running the registration is to resample the streamlines
 so that they both have the same number of points per streamline. Here we will
-use 20 points.
+use 20 points. This step is not optional. Inputing streamlines with different
```
Comment (arokem) [inline doc/examples/bundle_registration.py]: Typo: "Inputing" => "Inputting"

**Comment by arokem** [comment]
:+1:

**Comment by arokem** [comment]
This is all on the docs side, so I am not going to wait for Travis and go ahead with the merge.

**Pull request merged**